test/library/standard/Map/mapDefaultThresh.chpl

```diff
@@ -0,0 +1,4 @@
+use Map;
+
+var m: map(int, int);
+writeln(m.table.resizeThreshold);
```

Comment:
I'd maybe add a comment to these tests indicating that you're explicitly overriding the value, in case someone looks at them without noticing they have an execopts file